### PR TITLE
Chore: make the `basic-app` example up to date with our plugin scaffolding

### PR DESCRIPTION
**Pull request by leventebalogh**
**Related issue:** https://github.com/grafana/grafana-plugin-examples/issues/65

**What changed?**
We have recently updated the plugin scaffolding generated by https://github.com/grafana/create-plugin to contain a few fixes and additions (e.g. E2E testing). This PR updates the Basic App example to have all those changes.

### CODE REVIEW

**Review by mckn — APPROVED** [review]
LGTM!

File: examples/app-basic/package.json [hunk 1]
```diff
@@ -1,48 +1,67 @@
...
+    "e2e:open": "TZ=UTC grafana-e2e open",
+    "e2e:update": "TZ=UTC grafana-e2e run --update-screenshots",
+    "server": "docker-compose up -d --build"
   },
-  "author": "Your name",
+  "author": "Sample org",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@babel/core": "^7.16.7",
+    "@grafana/e2e": "latest",
```
[the remark this answers is not in the document]
Comment (leventebalogh) [reply examples/app-basic/package.json]: Good point! 👍 

Fixed in [2d49b3f](https://github.com/grafana/grafana-plugin-examples/pull/64/commits/2d49b3f1d31032f1a849336829b0db6a42a6cf88).

**Comment by tolzhabayev** [comment]
@leventebalogh looks like there are some merge conflicts

**Comment by leventebalogh** [comment]
Updated the example to use the latest tooling (using `@grafana/create-plugin migrate`) and also resolved the conflicts with the latest master.

**Review by jackw — APPROVED** [review]
🚀

**Pull request merged**